### ISSUE TITLE
upgrade future from 0.16.0 to 0.17.1 ?

### ISSUE THREAD

**Issue by abrahamvarricatt** [gadventures/gapipy#107]
Am attempting to install gapipy as part of another project. This comes up when running `pip install -r requirements.txt` there;

```
gapipy 2.19.3 has requirement future==0.16.0, but you'll have future 0.17.1 which is incompatible.
```

If it isn't too much trouble, can we consider upgrade future to version 0.17.1 ? (NOT an urgent priority)

**Comment by marz619** [comment]
Yup, its in our sights @abrahamvarricatt ... see https://github.com/gadventures/gapipy/commit/92d8d0003576855282516785e1d96f277f33d698
